xbmc/dialogs/GUIDialogContextMenu.cpp

```diff
@@ -224,20 +222,6 @@ bool CGUIDialogContextMenu::SourcesMenu(const std::string &strType, const CFileI
 
 void CGUIDialogContextMenu::GetContextButtons(const std::string &type, const CFileItemPtr& item, CContextButtons &buttons)
 {
-  // Add buttons to the ContextMenu that should be visible for both sources and autosourced items
-  if (item && item->IsRemovable())
-  {
-    if (item->IsDVD() || item->IsCDDA())
-    {
-      buttons.Add(CONTEXT_BUTTON_EJECT_DISC, 13391);  // Eject/Load CD/DVD!
```

Comment:
CONTEXT_BUTTON_EJECT_DISC is no longer used now and schould be removed from dialogs/GUIDialogContextMenu.h